meter/tasmota/types.go

```diff
@@ -85,3 +87,107 @@ type StatusSNSResponse struct {
 		}
 	}
 }
+
+func (v *StatusSNSResponse) UnmarshalJSON(data []byte) error {
```

Comment:
Das ist overkill. Du musst doch nur nur []float64 durch einen eigenen Typ ersetzen. Der könnte dann auch gleich eine `Value(channel id) (float64, error)` Funktion bekommen.

Reply:
Sorry, @andig , ich kapituliere. Ich glaube ich sehe den (go-)Wald vor lauter Bäumen nicht ... Du musst mir bitte auf die Sprünge helfen. :-)

Ich kann doch in dem eigenen Typen nicht float64 schon vordefinieren, sondern muss die Struct als Interface oder Byte Liste einlesen ...

Reply:
Dein eigener Typ wird einfach ein

    Channel []float64

ich mach dir einen Vorschlag, testen kann ichs aber nicht.